src/Servers/Connections.Abstractions/src/PublicAPI.Unshipped.txt

```diff
@@ -6,7 +6,7 @@ Microsoft.AspNetCore.Connections.Experimental.IMultiplexedConnectionBuilder.Use(
 Microsoft.AspNetCore.Connections.Experimental.IMultiplexedConnectionFactory
 Microsoft.AspNetCore.Connections.Experimental.IMultiplexedConnectionFactory.ConnectAsync(System.Net.EndPoint! endpoint, Microsoft.AspNetCore.Http.Features.IFeatureCollection? features = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionContext!>
 Microsoft.AspNetCore.Connections.Experimental.IMultiplexedConnectionListener
-Microsoft.AspNetCore.Connections.Experimental.IMultiplexedConnectionListener.AcceptAsync(Microsoft.AspNetCore.Http.Features.IFeatureCollection? features = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionContext!>
+Microsoft.AspNetCore.Connections.Experimental.IMultiplexedConnectionListener.AcceptAsync(Microsoft.AspNetCore.Http.Features.IFeatureCollection? features = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) -> System.Threading.Tasks.ValueTask<Microsoft.AspNetCore.Connections.Experimental.MultiplexedConnectionContext?>
```

Comment:
Nullable "breaking change" (although the package is experimental) @pranavkm

Reply:
Changes in Unshipped would be non-breaking.